### PR TITLE
fix: original link from WeChat and better performance

### PR DESCRIPTION
**Pull request by ChanceYu**

### CODE REVIEW

**Comment by kt286** [comment]
`decodeURIComponent(/"temp_url":"(.*)","source_url"/.exec(response)[1])`
就可以了  不需要 `puppeteer`

File: lib/routes/tencent/wechat/wemp.js [hunk 1]
```diff
@@ -37,16 +38,26 @@ module.exports = async (ctx) => {
...
 
-                const response = await ctx.cache.tryGet(link, async () => (await got.get(link)).data);
+                const response = await ctx.cache.tryGet(link, async () => {
+                    const browser = await puppeteer.launch();
+                    const page = await browser.newPage();
+                    await page.goto(link, { waitUntil: 'domcontentloaded' });
+                    const content = await page.content();
+                    await browser.close();
+                    return content;
+                });
```
Comment (kt286) [inline lib/routes/tencent/wechat/wemp.js]: decodeURIComponent(/"temp_url":"(.*)","source_url"/.exec(response)[1])

Comment (ChanceYu) [reply lib/routes/tencent/wechat/wemp.js]: 你调试一下就会发现，获取的是 `null`

Comment (kt286) [reply lib/routes/tencent/wechat/wemp.js]: /"temp_url":"(.*)","source_url"/.exec(response)[1])
/"url":"(.*)","temp_url"/.exec(response)[1])

两个其中一个有值

Comment (ChanceYu) [reply lib/routes/tencent/wechat/wemp.js]: 这种倒是也可以

Comment (kt286) [reply lib/routes/tencent/wechat/wemp.js]: 可能还有source_url
image

**Comment by kt286** [comment]
https://wemp.app/posts/3f5a98f5-f003-40a5-9e69-8ce398f8badc

**Comment by DIYgod** [comment]
Merged, thanks!

**Pull request merged**